pages/linux/unit-cli.md

```diff
@@ -0,0 +1,23 @@
+# unit-cli
+
+> CLI for unitcluster.com. Update code in local machine and launch it.
+
+- Start CLI with default or saved parameters:
+
+`unit-cli`
+
+- Create new unit:
+
+`new [name] [description]`
+
+- List units:
```

Comment:
I think this might be confused for the `ls` command in linux. Ping @agnivade thoughts?

Reply:
I believe this is the internal console command for `unit-cli`. Unless, the page's name is `ls`, I don't see much confusion. The description clearly says "List units".